### PR TITLE
Refactor header

### PR DESCRIPTION
**Pull request by mb-wali**
**1. bugfix(ui): home text & icon collapse in small screen.**

bug-header

* span tag for home icon.
* nowrap white-space css.

**2. refactor(UI): adding the text of 'Library & Archives'.**
Screenshot_2020-11-27 Repository

**3. logos and positioning.**
header-logos

**4. header - Todo**

@utnapischtim  any view?
aim_header

### CODE REVIEW

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/navbar.html [hunk 1]
```diff
@@ -65,6 +65,8 @@
             TU Graz
             <br>
             {{_ ("Repository")}}
+            <br>
+            {{_ ("Library & Archives")}}
```
Comment (utnapischtim) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/navbar.html]: I have only a comment for the commit message header. This is for me not a refactor type. The type refactor indicates that the result of the code is not changed and this is here the case. Therefore it is a feature for me. Further the part in which it is stated that the #110 is fixed should go in the footer of the commit body.

File: invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/footer.less [hunk 2]
```diff
@@ -48,4 +48,4 @@
  }
  .spacer-long{
   margin-top: 20px;
-}
\ No newline at end of file
+}
```
Comment (utnapischtim) [inline invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/footer.less:51]: This is for me not a bugfix, it is of the type style. It does not change the meaning of the code.

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/navbar.html [hunk 3]
```diff
@@ -62,11 +62,11 @@
...
         <div class="affiliation-text">
           <a title="RDM" href="{{url_for('invenio_theme_tugraz.index')}}">
-            TU Graz
+            TU GRAZ
             <br>
-            {{_ ("Repository")}}
+            {{_ ("REPOSITORY")}}
             <br>
-            {{_ ("Library & Archives")}}
+            {{_ ("LIBRARAY & ARCHIVES")}}
```
Comment (utnapischtim) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/navbar.html]: It is difficult here what type this is. It is not a refactor type for me, but it is also not really of the type feature. Maybe we should add a new type like "modification"?

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/navbar.html [hunk 4]
```diff
@@ -66,7 +66,7 @@
             <br>
             {{_ ("REPOSITORY")}}
             <br>
-            {{_ ("LIBRARAY & ARCHIVES")}}
+            {{_ ("LIBRARY & ARCHIVES")}}
```
Comment (utnapischtim) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/navbar.html:69]: That would be of the type "typo" for me.

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/navbar.html (hunk 1 above)
Comment (mb-wali) [reply invenio_theme_tugraz/templates/invenio_theme_tugraz/navbar.html]: good to know.

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/navbar.html (hunk 3 above)
Comment (mb-wali) [reply invenio_theme_tugraz/templates/invenio_theme_tugraz/navbar.html]: sounds good.

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/header.html [hunk 5]
```diff
@@ -32,6 +33,7 @@
                   </polyline>
                 </svg>
                 {{_ ("Home")}}
+              </span>
```
Comment (utnapischtim) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/header.html:36]: Style: Normally those additions should be indented correctly.
Bugfix: Thumbs UP

**Comment by utnapischtim** [comment]
About the DE/EN topic: how should we do if we have more languages? Like DE/EN/FR/ES/IT and so on?

Maybe we should do it as a drop down if there are more then 2 options and only show the current style. We should postpone that part. Create an issue for future thoughts.

But for now, we should do it like tugraz.at does. Only DE or EN. So if the page is set to German we display EN and otherwise the other way around. So we are consistent with tugraz.at

**Comment by mb-wali** [comment]
### questions
@utnapischtim 
1. which below texts should we use:
* ```login``` or ```log in```?
* ```sign in``` or ```signin```?
* ```log out``` or ```logout```?
* ```sign up``` or ```signup```?

2. what do you think if we remove the icons from login and signup buttons? since other TU Graz web pages does not have it . and also i think it will look cleaner.

3. should we use the language icon like ```DE/EN``` instead of ```DE``` and toggle ```EN```?

**Comment by mb-wali** [comment]
> About the DE/EN topic: how should we do if we have more languages? Like DE/EN/FR/ES/IT and so on?
> 
>Maybe we should do it as a drop down if there are more then 2 options and only show the current style. We should postpone that part. Create an issue for future thoughts.
> 
> But for now, we should do it like tugraz.at does. Only DE or EN. So if the page is set to German we display EN and otherwise the other way around. So we are consistent with tugraz.at

ok - that sound good.

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/header.html (hunk 5 above)
Comment (mb-wali) [reply invenio_theme_tugraz/templates/invenio_theme_tugraz/header.html:36]: duly noted.

**Comment by utnapischtim** [comment]
>     1. which below texts should we use:
> 
>     * `login` or `log in`?
>     * `sign in` or `signin`?
>     * `log out` or `logout`?
>     * `sign up` or `signup`?

we should go with `Log in` for logging in to existing account
we should go with `Log out` to log out from existing account
we should go with `Sign up` to register a new user. (this would start a automatic process for shibboleth and a manually register for a external user)

Why `Log in` over `Sign in`. For me as a non native speaker it is more intuitive to have the log in button more separated from the register button. `Sign in` and `Sign up` are for me to equal.

To be consistent it is then also better to use `Log out` because we have the `Log in` button.

**Comment by utnapischtim** [comment]
>     1. what do you think if we remove the icons from login and signup buttons? since other TU Graz web 
>        pages does not have it . and also i think it will look cleaner.

yep, remove it.

**Pull request merged**